lib/Yancy/Help/Config.pod

```diff
@@ -425,6 +425,18 @@ Instead of a filter name, you can provide an array. The first member
 will be the name, and any further members will be passed to the filter
 code-ref as parameters after the mandatory three.
 
+=item x-view-of
```

Comment:
Add some examples of why one might use this.

This looks very much like a step on the path to "virtual collections" (which is this + a search filter). Would it be better to just call this `x-view` and then allow a single string to be the collection, or (in a future commit) a hashref of data like:

```json
{
    "collections": {
        "admins": {
            "x-view": {
                "collection": "users",
                "search": { "is_admin": true }
            }
        }
    }
}
```

"properties" could then be optional, with the default value being all the same properties of the underlying real collection.

Over time we could then even make this virtual collection editable :)

Reply:
I'm pleased to switch to `x-view` be an object with one key `collection`, and to add an example.

Reply:
Excellent. That'll be the last thing and then we can get this out the door. No rush, can't rightly rush y'all when it took this long for me to spend 30 minutes combing through it 😛

Reply:
All done! Please take a look.

By the way, welcome back ;-)